.github/workflows/weekly_pull_requests.yaml

```diff
@@ -7,6 +7,10 @@ on:
 
 jobs:
     weekly_pull_requests:
+
+        # dont run jobs on forks, because gh-tokens are not available 
+        if: github.event.pull_request.head.repo.full_name == github.repository
+
```

Comment:
This whole workflows should only run on main repository.

Can you move the `if` to first level to make sure no other jobs will be run? As `on` is?

Reply:
not sure what you mean. I am not aware of the place where a `if:` can be placed above this per-job level.

Reply:
This one:
https://github.community/t/conditional-jobs-and-workflows/16296/10

It seems it's not possible :/